pyat/setup.py

```diff
@@ -81,40 +79,30 @@ def select_omp():
 if not sys.platform.startswith('win32'):
     cflags += ['-Wno-unused-function']
```

Comment:
Commenting on this bit so I can have a thread, but this is for line 31.

I find I need to use:
`cflags = ['--std=c99']`

Otherwise, I get an error about C99 `for` loops. This was not a problem in 0.2.1.

Reply:
Very strange… I does not happen in GitHub actions. Which platform are you using ?

Reply:
The following fails on RHEL7:
`gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC)`

The following works on RHEL8:
`gcc version 8.5.0 20210514 (Red Hat 8.5.0-4) (GCC) `

As far as I can tell, it is a valid complaint:
```
    integrator-src/WakeFieldPass.c:132:9: error: ‘for’ loop initial declarations are only allowed in C99 mode
             for(int i=0;i<3;i++){
             ^
    integrator-src/WakeFieldPass.c:132:9: note: use option -std=c99 or -std=gnu99 to compile your code
```

Reply:
I see. WakeFieldPass.c was recently introduced, so you had no problem before. Apparently, all recent compilers are by default compliant with c99, only your very old one is not (gcc 4 !!). Once it's understood, I have no objection to add -std=c99.